Power Up Proto/src/org/usfirst/frc/team5493/robot/commands/EncoderForDistance.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team5493.robot.commands;
+
+import org.usfirst.frc.team5493.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class EncoderForDistance extends Command {
+	
+	private double target;
+	private double speed;
+	private boolean goForward;
+
+    public EncoderForDistance(double distance, boolean forward) {
+          requires(Robot.driveBase);
+          if(forward)
+        	  target = distance;
+          else
+        	  target = -distance;
+          goForward = forward;
+          
+        
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() { 
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)